model/time_series_update.go

```diff
@@ -128,25 +167,32 @@ func GetPerformanceResultSeriesIdsNeedingTimeSeriesUpdate(ctx context.Context, e
 		},
 		{
 			"$replaceRoot": bson.M{
-				"newRoot": "$_id",
+				"newRoot": bson.M{
+					unanalyzedPerformanceSeriesProjectKey:      "$" + bsonutil.GetDottedKeyName("_id", unanalyzedPerformanceSeriesProjectKey),
+					unanalyzedPerformanceSeriesVariantKey:      "$" + bsonutil.GetDottedKeyName("_id", unanalyzedPerformanceSeriesVariantKey),
+					unanalyzedPerformanceSeriesTaskKey:         "$" + bsonutil.GetDottedKeyName("_id", unanalyzedPerformanceSeriesTaskKey),
+					unanalyzedPerformanceSeriesTestKey:         "$" + bsonutil.GetDottedKeyName("_id", unanalyzedPerformanceSeriesTestKey),
+					unanalyzedPerformanceSeriesArgumentsKey:    "$" + bsonutil.GetDottedKeyName("_id", unanalyzedPerformanceSeriesArgumentsKey),
+					unanalyzedPerformanceSeriesMeasurementsKey: "$" + unanalyzedPerformanceSeriesMeasurementsKey,
+				},
 			},
 		},
 	})
 	if err != nil {
 		return nil, errors.Wrapf(err, "getting metrics needing change point detection")
 	}
 	defer cur.Close(ctx)
-	var res []PerformanceResultSeriesID
+	var res []UnanalyzedPerformanceSeries
 	err = cur.All(ctx, &res)
 	if err != nil {
 		return nil, errors.Wrapf(err, "decoding metrics needing change results")
 	}
 	return res, nil
 }
 
-// GetPerformanceResultSeriesIDs finds all performance result series IDs in the
-// the database.
-func GetPerformanceResultSeriesIDs(ctx context.Context, env cedar.Environment) ([]PerformanceResultSeriesID, error) {
+// GetAllPerformanceResultSeriesIDs finds all performance result series in the
+// database.
+func GetAllPerformanceResultSeriesIDs(ctx context.Context, env cedar.Environment) ([]UnanalyzedPerformanceSeries, error) {
```

Comment:
* Should this be tested?
* This query is pretty similar to `GetUnanalyzedPerformanceSeries`, which has limits to avoid excessive query runtime. Wouldn't this be more expensive since it doesn't limit the results?

Reply:
Yeah this is aggregation would only be used in an emergency situation where we would want to rerun all of the SPS notifications, it has never been used AFAIK. It has to be manually called via a protected REST route. It is tested in the `rest` package.